SAP2000_Engine/SAP2000_Engine.csproj

```diff
@@ -95,6 +95,10 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\Results_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="SAP2000_oM">
+      <HintPath>C:\ProgramData\BHoM\Assemblies\SAP2000_oM.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
```

Comment:
This shouldn't be necessary at all with the project reference? Does it not work without this?

Reply:
```suggestion
```